geomesa-features/geomesa-feature-common/src/main/scala/org/locationtech/geomesa/features/serialization/GeoJsonSerializer.scala

```diff
@@ -88,6 +88,8 @@ class GeoJsonSerializer(sft: SimpleFeatureType) extends LazyLogging {
 
 object GeoJsonSerializer extends LazyLogging {
 
+  import org.locationtech.geomesa.utils.geotools.ObjectType
```

Comment:
Why is this import needed?

Reply:
i moved ObjectType out of the geomesa-features-kryo and into geomesa-utils so that i could use it in arrow

Reply:
Sorry, I should I mentioned that I was asking since the same import is at the top of the file.

Reply:
ah, I could probably remove the top-level one now. the old ObjectTypes (which is still there but deprecated) is in this same package, and it seemed to override the top-level import so i had to also add it here to get it to compile. let me try removing the other one.

Reply:
oh, the top-level on is the type ObjectType.ObjectType, while this one is the object ObjectType itself. they do both need to be there for it to compile. i moved the top-level one down here for consistency.

Reply:
Ah!  *blinks*  I cannot read.  Thank you!